packages/asset/stories/asset.stories.ts

```diff
@@ -23,7 +23,7 @@ export default {
 export const Default = (): TemplateResult => {
     return html`
         <sp-asset style="height: 128px">
-            <img src=${portrait} alt="Demo Image" />
+            <img src=${portrait} alt="Demo Graphic" />
```

Comment:
Everywhere I switch `Image` to `Graphic` is in a demo or test, but it seems like good practice to get into.